articles/integrations/azure-api-management/configure-azure.md

```diff
@@ -0,0 +1,172 @@
+
+![](/media/articles/integrations/azure-api-mgmt/azure/api-mgmt-service-config.png)
+
+Click **Create** to begin provisioning your service.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/deployment-in-progress.png)
+
+### Import Your API
+
+::: note
+For this tutorial, we will be importing and using the Calculator API provided by Microsoft. You can, however, create your own API instead of using the Calculator API.
+:::
+
+Launch the API Management service that you created in the previous step. 
+
+![](/media/articles/integrations/azure-api-mgmt/azure/api-mgmt-service-home.png)
+
+Open up the Publisher Portal, and click on **Import API**.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/publisher-portal.png)
+
+You'll be importing an API **from URL**. 
+
+![](/media/articles/integrations/azure-api-mgmt/azure/import-api.png)
+
+To do so, provide values for the following parameters:
+
+| Parameter | Description |
+| --------- | ----------- |
+| Specification document URL | the URL Azure will use to retrieve your API's specification. For this example, use `http://calcapi.cloudapp.net/calcapi.json`. |
+| Specification format | the API specification format. Use `Swagger`. |
+| New/Existing API | set this to **New** |
+| Web API URL suffix | the value appended to the base URL of your API management service that uniquely identifies the API you're currently creating, such as `calc` |
+| Web API URL scheme | the protocol used to access your API (for this example, set this to `HTTPs`) |
+| Products | Add this API to the `Starter` product. This is a basic, getting-started-with-Azure container that holds your sample products and applies entry-level rate limits to your calls. |
+
+![](/media/articles/integrations/azure-api-mgmt/azure/import-api-config.png)
+
+When done, click **Save** to import your API. You'll be redirected to the summary page for your API when it's fully imported.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/basic-calc-api.png)
+
+### Configure Your OAuth 2.0 Authorization Server
+
+To use Auth0 to secure your API, you'll need to register Auth0 as an OAuth 2.0 Authorization Server. You can do so using the Azure Publisher Portal.
+
+Navigate to **Security** > **OAuth 2.0**.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/oauth2-servers.png)
+
+Click on **Add Authorization Server**. You'll see the configuration screen that lets you provide details about your Auth0 account.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/new-oauth2-server-config.png)
+
+::: note
+For the purposes of this example, we'll use the **Authorization Code grant type**, but you're free to use whichever grant type is most appropriate for your use case. Azure currently supports the following grant types: Authorization Code, Implicit, Resource Owner Password, Client Credentials.
```

Comment:
I'm a bot confused here, we will use the Authorization Code Grant while we created a non interactive client? I expected to see Client Credentials here.

Also, add links to docs.